mmcv/runner/base_module.py

```diff
@@ -47,8 +47,8 @@ def init_weight(self):
                     module.init_weight()
             self._is_init = True
         else:
-            warnings.warn('This module has bee initialized, \
-                please call initialize(module, init_cfg) to reinitialize it')
+            warnings.warn(f'init_weight of {self.__class__.__name__} has '
+                          f'been called more than once.')
```

Comment:
Agree, but "please call initialize(module, init_cfg) to reinitialize it"  should be kept to give a tip for users what they could do.

Reply:
Actually, I am confused about why we should reinitialize it?

Reply:
It should not be initialized again, so I leave a warning as a reminder.

Reply:
So  what dose ```please call initialize(module, init_cfg) to reinitialize it``` mean?